dot/parachain/collator-protocol/validator_side.go

```diff
@@ -499,6 +748,7 @@ func (cpvs CollatorProtocolValidatorSide) handleNetworkBridgeEvents(msg any) err
 		// TODO #4155
 	case networkbridgeevents.OurViewChange:
 		// TODO #4156
+		return cpvs.handleOurViewChange(msg.View)
```

Comment:
Should above TODO #4156 comment be removed since this PR handle that?

Reply:
removed it